infra/function.tf

```
@@ -0,0 +1,71 @@
+data "archive_file" "source" {
+  type        = "zip"
+  source_dir  = "${path.module}/../"
+  output_path = "/tmp/zombie-watcher-source.zip"
+  excludes = [
+    "terraform/",
```

Change excludes pattern to match `infra` folder instead of `terraform`.

```suggestion
    "infra/",
```

---

Fixed!